### PR TITLE
fix: config files

### PR DESCRIPTION
**Pull request by 0xDiscotech**
The `deployConfig/shared.ts` file was updated with the new format Connext defined in their repo, and also the `.github/workflows/build-test-deploy.yml`, `ops/testnet/staging/core/main.tf`, `package.json` and `packages/deployments/contracts/.env.example` files were modified to be the equal than the `main` branch files.

### CODE REVIEW

**Review by ashitakah — APPROVED** [review]
🔥

File: ops/testnet/staging/core/main.tf [hunk 1]
```diff
@@ -304,7 +304,7 @@ module "lighthouse_prover_cron" {
   })
   schedule_expression    = "rate(10 minutes)"
   timeout                = 300
-  memory_size            = 2048
+  memory_size            = 512
```
Comment (0xDiscotech) [inline ops/testnet/staging/core/main.tf:307]: Restated to the value that they have on their main branch.

File: ops/testnet/staging/core/main.tf [hunk 2]
```diff
@@ -361,7 +361,7 @@ module "lighthouse_process_from_root_cron" {
   stage               = var.stage
   container_env_vars  = merge(local.lighthouse_env_vars, { LIGHTHOUSE_SERVICE = "process" })
   schedule_expression = "rate(5 minutes)"
-  memory_size         = 2048
+  memory_size         = 1536
```
Comment (0xDiscotech) [inline ops/testnet/staging/core/main.tf:364]: Restated to the value that they have on their main branch.

File: package.json [hunk 3]
```diff
@@ -42,8 +42,6 @@
   "scripts": {
     "lint:all": "yarn workspaces foreach --parallel --exclude @connext/nxtp-subgraph --exclude @connext/bridge-reference --exclude @connext/nxtp-integration --exclude @connext/smart-contracts run lint",
     "test:all": "yarn workspaces foreach --parallel --exclude @connext/nxtp-relayer --exclude @connext/nxtp-subgraph --exclude @connext/nxtp-integration --exclude @connext/sdk run test",
-    "test:contracts": "yarn workspace @connext/smart-contracts run test",
-    "test:connectors": "yarn workspace @connext/smart-contracts run test:connectors",
```
Comment (0xDiscotech) [inline package.json:46]: Remove connectors and contracts scripts since they are no more necessary and they don't exist on their main branch.

**Pull request merged**